app/controllers/admin/creator_settings_controller.rb

```diff
@@ -1,15 +1,26 @@
 module Admin
   class CreatorSettingsController < Admin::ApplicationController
-    ALLOWED_PARAMS = %i[community_name logo_svg primary_brand_color_hex invite_only_mode public checked_code_of_conduct
-                        checked_terms_and_conditions].freeze
+    ALLOWED_PARAMS = %i[community_name primary_brand_color_hex invite_only_mode public checked_code_of_conduct
+                        checked_terms_and_conditions logo].freeze
 
-    def new; end
+    def new
+      @max_file_size = LogoUploader::MAX_FILE_SIZE
+      @logo_allowed_types = (LogoUploader::CONTENT_TYPE_ALLOWLIST +
+        LogoUploader::EXTENSION_ALLOWLIST.map { |extension| ".#{extension}" }).join(",")
+    end
 
     def create
       extra_authorization
+
+      raise CarrierWave::IntegrityError if settings_params[:logo].blank?
```

Comment:
Is uploading a logo mandatory task on setup? wouldn't it be better if this is something we allow forem-creator to circle back when if they don't already have one prepared?

Reply:
@maestromac this is a very good point, thank you for bringing it up. I confirmed with product and will remove the validation and amend the code accordingly.

Reply:
✅ done, I've removed the requirement for logo.